### PR TITLE
change dependencies of `keccak-hash`and `trie-standardmap`

### PR DESCRIPTION
**Pull request by koushiro**
Change  `ethereum-types` dep to `primitive-types` dep of `keccak-hash`.
Remove the `ethereum-types` dep of `trie-standardmap`.

Need to process the  #110 firstly.

### CODE REVIEW

File: fixed-hash/src/hash.rs [hunk 1]
```diff
@@ -51,7 +51,7 @@ macro_rules! construct_fixed_hash {
 	( $(#[$attr:meta])* $visibility:vis struct $name:ident ( $n_bytes:expr ); ) => {
 		#[repr(C)]
 		$(#[$attr])*
-		$visibility struct $name ([u8; $n_bytes]);
+		$visibility struct $name (pub [u8; $n_bytes]);
```
Comment (Robbepop) [inline fixed-hash/src/hash.rs]: I think this line is already part of the other PR, isn't it?

Comment (koushiro) [reply fixed-hash/src/hash.rs]: yes, it is.

**Comment by koushiro** [comment]
@Robbepop  #110 has been merged, PTAL this PR again.

**Comment by dvdplm** [comment]
@koushiro seems like the `triehash` tests hang on windows for some reason. Any idea what is going on there?

**Comment by koushiro** [comment]
> @koushiro seems like the `triehash` tests hang on windows for some reason. Any idea what is going on there?

#118

**Comment by ordian** [comment]
a rebase on master should help

**Comment by koushiro** [comment]
@dvdplm PTAL again.

**Pull request merged**